### PR TITLE
Initialisation with list of paths

### PR DESCRIPTION
**Pull request by frederickfogerty**
This PR allows `command(['scripts/**/*.cmd.ts', 'otherDir/scripts/**/*.cmd.ts]')`

### CODE REVIEW

File: src/init.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ function toModulePaths(param: string): Array<string> {
       .map(p => fsPath.join(dir, p));
 
     // Just the JS files.
-    paths = items.filter(p => p.endsWith('.js'));
+    paths = items.filter(p => p.endsWith('.js') || p.endsWith('.ts'));
```
Comment (frederickfogerty) [inline src/init.ts:29]: Supports `.ts` cmd files (such as `deploy.cmd.ts`)

File: src/init.ts [hunk 2]
```diff
@@ -68,8 +68,8 @@ function toCommand(modulePath: string): ICommand {
     description: m.description,
     group: m.group,
     args: m.args,
-    validate: <IValidate> m.validate,
-    action: <IAction> action,
+    validate: m.validate as IValidate,
+    action: action as IAction,
```
Comment (philcockfield) [inline src/init.ts:72]: 👍

File: src/init.ts [hunk 3]
```diff
@@ -92,14 +92,26 @@ function toCommands(modulePaths: Array<string>) {
 }
 
 
+function pathToCommands(path: string) {
+  const paths = toModulePaths(path);
+  return toCommands(paths);
+
```
Comment (philcockfield) [inline src/init.ts]: white space!

File: src/init.ts [hunk 4]
```diff
@@ -92,14 +92,26 @@ function toCommands(modulePaths: Array<string>) {
...
+}
 
 
-export default (param) => {
+export default (param: string | string[] | { [key: string]: ICommand }) => {
```
Comment (philcockfield) [inline src/init.ts:103]: 🌮

**Review by philcockfield — COMMENTED** [review]
👍  LGTM

**Review by philcockfield — APPROVED** [review]

**Pull request merged**